### PR TITLE
fix for kops certificate update error ...

### PR DESCRIPTION
**Pull request by prageethw**
#20540

### CODE REVIEW

**Comment by prageethw** [comment]
@googlebot I signed it!

File: install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml [hunk 1]
```diff
@@ -134,6 +134,7 @@ data:
       scheme: https
       tls_config:
         ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        server_name: kubernetes
```
Comment (elfinhe) [inline install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml]: we are moving away from the old helm charts, and planning to use https://github.com/istio/istio/tree/master/manifests with `istioctl manifest apply`.

do you know if this issue also exists in https://github.com/istio/istio/tree/master/manifests?

Comment (prageethw) [reply install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml]: yep, even I installed using istioctl manifest apply, I thought istioctl internally talks to helm charts to install add-ons such as Prometheus, hence I made the PR, yeah if that is not the case this exist in manifest too. :) I will make another PR

**Review by johscheuer — COMMENTED** [review]
It seems like you need to run `make gen` to update `operator/pkg/vfs/assets.gen.go`

**Comment by prageethw** [comment]
> It seems like you need to run `make gen` to update `operator/pkg/vfs/assets.gen.go`

TBH I don't know how to do this, when I th

**Pull request closed without merging**